packages/netlify-cms-backend-github/src/API.js

```diff
@@ -1,6 +1,7 @@
 import { Base64 } from 'js-base64';
 import semaphore from 'semaphore';
-import { find, flow, get, hasIn, initial, last, partial, result, uniq } from 'lodash';
+import { find, flow, get, hasIn, initial, last, partial, result, differenceBy } from 'lodash';
+import trimStart from 'lodash/trimStart';
```

Comment:
Should we include this in the line above and let treeshaking resolve the rest? Might need the lodash webpack plugin for that to actually happen.

Reply:
You're right. I probably copy pasted it from somewhere and didn't notice the existing import. Going forward we can use https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/import-scope.md to enforce a way to import lodash methods (probably `Destructured Members`) and use a Webpack plugin to get tree shaking like you said.